pages/success.tsx

```diff
@@ -157,9 +157,28 @@ export async function getServerSideProps(context) {
         }
     });
 
+    const subEventTypes = user.eventTypes.map(el => {
+        if (el.startDate !== null) {
+          return {
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/calendso/calendso/pullRequest?prid=7689307)
```suggestion
  return {
```